### ISSUE TITLE
Player renders twice in strict mode

### ISSUE THREAD

**Issue by mikehilario** [brightcove/react-player-loader#102]
![Screen Shot 2022-09-06 at 4 15 43 PM](https://user-images.githubusercontent.com/6709776/188730809-2693490e-5db7-410b-ade0-6dbe4e7e90a3.png)

Player renders twice in strict mode.  Please see codesandbox: https://codesandbox.io/s/ml6vzj

**Comment by jhopley-stadion** [comment]
Yea i am getting the same issue? Did you resolve this?

**Comment by mikehilario** [comment]
Unfortunately, no. @jhopley-stadion

**Comment by jhopley-stadion** [comment]
Nightmare! Is anyone supporting this repo?

**Comment by SuperCrunchTime** [comment]
Also having this issue.

**Comment by ganapathyraiker** [comment]
Hello, are there any solution for this observation ?

**Comment by Unicornelia** [comment]
I am also fighting this issue, this is a nightmare, anybody any suggestions?

**Comment by labgit771** [comment]
> I am also fighting this issue, this is a nightmare, anybody any suggestions?
This works:

  <ReactPlayerLoader
            accountId="1752604059001"
            videoId="5819230967001"
            playerId="default"
            embedType="iframe"
        />

**Comment by labgit771** [comment]
Render it as type iframe, it works..

**Comment by Unicornelia** [comment]
> Render it as type iframe, it works..
> 
> embedType='iframe'

I think it depends on how the video was uploaded, that is why the iframe works, as long as I understood the docs

**Comment by labgit771** [comment]
> > Render it as type iframe, it works..
> > embedType='iframe'
> 
> I think it depends on how the video was uploaded, that is why the iframe works, as long as I understood the docs

Yeah, but sometime as a dev we don't have access to BrightCode account to manage video to see how it was uploaded.  But yeah , i agree with you !

**Comment by dannydebug** [comment]
This issue got resolved in the new published version 1.5.0.

**Comment by rsauchuck-gp** [comment]
Using version 1.5.0 I randomly get an empty div the same style as the div wrapping the video player
```
<div style="padding-top: 177.778%;"></div>
<div style="padding-top: 177.778%;"><video-js></video-js><div>
```

When the guard triggers correctly I do not get the extra div
